src/analytics/AppAnalytics.ts

```diff
@@ -125,9 +125,7 @@ class AppAnalytics {
         const deviceInfo = await getDeviceInfo()
         this.deviceInfo = deviceInfo
         uniqueID = deviceInfo.UniqueID
-        this.sessionId = sha256FromString('0x' + uniqueID.split('-').join('') + String(Date.now()))
-          .toString('hex')
-          .slice(2)
+        this.sessionId = sha256(Buffer.from(uniqueID + String(Date.now()))).slice(2)
```

Comment:
do we still need to remove the hypens from `uniqueID`?

Reply:
No it's not needed anymore